documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,95 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes methodology of Knotx application performance tests.
+
+## Goals
+Goals of conducted tests are as following:
+* Assess system performance,
+* Assess system stability.
+
+### Metrics
+During our tests we focus on following metrics and correlate it to came out with observations:
```

Comment:
> During our tests we focus on following metrics and correlate it to came out with observations:

This sentence is grammatically incorrect and I'm not sure what you meant to say. I'll be happy to help you rephrase it if drop me a line on Skype.